packages/core/src/tile/model/geometry/geometryManager.ts

```diff
@@ -93,13 +93,33 @@ export class GeometryManager extends Manager<
       this.scene.getEngine().getRenderHeight()
     );
 
+    this.initializeUniformBuffer();
+
     this.registerEventListeners();
   }
 
   public get CRS(): string | undefined {
     return this.metadata.crs;
   }
 
+  private initializeUniformBuffer() {
+    this.polygonOptions = new UniformBuffer(this.scene.getEngine());
+
+    this.polygonOptions.addUniform('color', 4, 0);
+    this.polygonOptions.addUniform('opacity', 1, 0);
+    this.polygonOptions.addUniform('colorScheme', 4, 32);
```

Comment:
Are these related to the 32 & 192 I asked about above? If so should they be named constants somewhere?